runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -953,6 +962,54 @@ class VM_JFRConstantPoolTypes {
 		j9mem_free_memory(getJFRConstantEvents(vm)->OSInfoEntry.osVersion);
 	}
 
+	/**
+	 * Initialize InitialSystemProperty.
+	 *
+	 * @param vm[in] the J9JavaVM
+	 */
+	static void initializeInitialSystemProperty(J9JavaVM *vm)
```

Comment:
Do we actually need to save the sys props? could you not just walk the pool in `writeInitialSystemPropertyEvents` ?

Reply:
That makes sense. Maybe the same argument could also apply to other constant events since we're only writing them once?